packages/space-opera/src/components/mobile_view/mobile_view.ts

```diff
@@ -182,23 +221,51 @@ export class MobileView extends ConnectedLitElement {
           min-field-of-view=${ifDefined(config.minFov)}
           max-field-of-view=${ifDefined(config.maxFov)}
           animation-name=${ifDefined(config.animationName)}
+          @load=${this.newModel}
         >${childElements}</model-viewer>
       </div>
     </div>`;
   }
 
+  /**
+   * Determine the mobile operating system.
+   * This function returns one of 'iOS', 'Android', 'Windows Phone', or
+   * 'unknown'.
+   * https://stackoverflow.com/questions/21741841/detecting-ios-android-operating-system
+   */
+  getMobileOperatingSystem(): string {
+    // @ts-ignore
+    var userAgent = navigator.userAgent || navigator.vendor || window.opera;
```

Comment:
Why is TS annoyed about this? And how about `const` instead of `var`?

Reply:
I'll change that to const in the next PR where I clean up these features code-wise a bit more.

And the window.opera wasn't recognized which is why the ts-ignore is there.